src/sentry/static/sentry/app/views/onboarding/projectSetup/firstEventIndicator.tsx

```diff
@@ -10,16 +9,21 @@ import EventWaiter from 'app/views/onboarding/projectSetup/eventWaiter';
 import InlineSvg from 'app/components/inlineSvg';
 import space from 'app/styles/space';
 import testablePose from 'app/utils/testablePose';
+import {Group, Organization} from 'app/types';
 
-const FirstEventIndicator = props => (
+type EventWaiterProps = Omit<React.ComponentProps<typeof EventWaiter>, 'children'>;
```

Comment:
@EvanPurkhiser Do we really need all the React component props to use this component? You can't just use `EventWaiter['props']`?

Reply:
Unfortunately that does not work, since EventWaiter is exported wrapped with the withApi HoC